### PR TITLE
[import wizard 2] Upload and info pages

### PR DESCRIPTION
**Pull request by chejennifer**
- add relevant files for the upload and info pages
- navigation and integration with the rest of the tool will be in future PR
screen-recording-_48__1

### CODE REVIEW

File: static/css/import_wizard2.scss [hunk 1]
```diff
@@ -14,4 +14,126 @@
  * limitations under the License.
  */
 
-@import "base";
\ No newline at end of file
+@import "base";
+
+#main-body {
+  position: relative;
```
Comment (juliawu) [inline static/css/import_wizard2.scss:20]: Style suggestion: [Google's CSS Style Guide](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order) suggests alphabetizing declarations.

**Review by juliawu — APPROVED** [review]
LGTM, @beets may have more comments.

File: static/css/import_wizard2.scss (hunk 1 above)
Comment (chejennifer) [reply static/css/import_wizard2.scss]: done

**Comment by chejennifer** [comment]
thanks for the review! Will merge now & if there are additional comments from @beets, I can open a new PR to address them

**Pull request merged**